src/components/delegates/delegates.js

```diff
@@ -54,6 +56,39 @@ app.component('delegates', {
       this.$scope.$on('peerUpdate', () => {
         this.updateAll();
       });
+
+      const $ctrl = this;
```

Comment:
This part is inconsistent with the rest of the code which uses `this`. may be using `function.bind` solves this issue.